src/main/java/edu/virginia/lib/fedora/aview/AView.java

```
@@ -42,31 +47,50 @@ public AView() throws URISyntaxException, MalformedURLException {
         accessionsTemplate = ve.getTemplate("accessions.vm");
         bagsTemplate = ve.getTemplate("bags.vm");
         filesTemplate = ve.getTemplate("files.vm");
+        
+        try {
+        	BufferedReader br = new BufferedReader(new FileReader("host.txt"));
+        	host = br.readLine();
+        	br.close();
+        	
+        	if (host.equals(""))
+        		host = null;
+        } catch (IOException e) {
```

Just let this be thrown by the constructor if it occurs.